components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -4952,12 +4959,20 @@ def getBody(self):
 
         namedGetter = self.descriptor.operations['NamedGetter']
         if namedGetter:
-            named = ("if RUST_JSID_IS_STRING(id) && !has_property_on_prototype(cx, proxy, id) {\n" +
-                     CGIndenter(CGProxyNamedGetter(self.descriptor)).define() + "\n" +
-                     "    *bp = found;\n"
-                     "    return true;\n"
-                     "}\n" +
-                     "\n")
+            named = """\
+if RUST_JSID_IS_STRING(id) {
+    let mut has_on_proto = false;
+    if !has_property_on_prototype(cx, proxy, id, &mut has_on_proto) {
+        return false;
```

Comment:
Shouldn't this set `*bp = false`? (sorry if it's a dumb question, I was digging into the reason for #13165 and arrived here).